README.md

```diff
@@ -1,8 +1,8 @@
-# Generic New Project Template
+# Generic New Project Template <!-- omit in toc -->
```

Comment:
I find the HTML comment ugly and hacky, but am willing to live with it, for now.

Reply:
There is a way to do it with a separate, root-level, markdown config file. I agree it looks ugly but it the idea of using comments as directives in other languages is not unusual.

Reply:
I think it is just because it is the title, if it were anywhere else on the page, the comments would feel more natural.  I still think this is better than adding yet another config file to the root.